### PR TITLE
Remove system-config

### PR DESCRIPTION
**Pull request by jpdagostino**

### CODE REVIEW

**Comment by jpdagostino** [comment]
Update to #667

File: system/install.sh [hunk 1]
```diff
@@ -6,46 +6,34 @@ fi
 
 cd /
 echo Updating Debian
-apt-get update &&
+apt-get update
 apt-get upgrade -y
-apt-get install -y curl nano sudo git build-essential
+
+echo Installing Extras
+apt-get install -y curl nano git build-essential
```
Comment (Jack-Papel) [inline system/install.sh:13]: This should probably be in the install deps script. Or at least curl should be - it's not used in this script

**Review by Jack-Papel — APPROVED** [review]
Address comments

**Comment by jpdagostino** [comment]
Addressed comments 👍

**Pull request merged**